contrib/cirrus/logformatter

```diff
@@ -418,10 +418,13 @@ END_HTML
         }
     }
 
-    # FIXME: if Cirrus magic envariables are available, write a link to results
+    # If Cirrus magic envariables are available, write a link to results.
+    # FIXME: it'd be so nice to make this a clickable live link.
+    # Magic Blob can be obtained from:
+    #   https://console.cloud.google.com/storage/browser?project=libpod-218412
```

Comment:
How about we just document this here.  Something like:

```
    # Note: The value for STATIC_MAGIC_BLOB is the name of a google-storage bucket.
    # They can be obtained from: https://console.cloud.google.com/storage/browser?project=libpod-218412
    # The bucket is created on first-use by Cirrus-CI when setting up a new repository, and will remain
    # for the life of the project.  Bucket names must be globally-unique, so Cirrus-CI appends it's internal
    # ID for the repository and a unique nonce (maybe the github project ID?).
    # By default, these buckets are set private.  However since the contents
    # are public (due to this being an OSS repository), it's fine to make the bucket
    # public.  This is done by adding an entry via the google-storage WebUI:
    # https://console.cloud.google.com/storage/browser/<STATIC_MAGIC_BLOB>?project=<PROJECT_ID>
    # under the 'permissions' tab, for example:
    #     Members: 'allUsers'; Role(s): 'Storage Object Viewer'.
```

Reply:
Thanks. If tests pass, I'll update the comment and push again.